lib/web_ui/lib/src/engine/canvaskit/color_filter.dart

```diff
@@ -81,8 +81,26 @@ class CkBlendModeColorFilter extends CkColorFilter {
   final ui.Color color;
   final ui.BlendMode blendMode;
 
+  static Float32List get _identityMatrix {
```

Comment:
rather than recreating the identity matrix for every call you could late initialize it like

```
static Float32List identityTransform => _identityTransform ?? _computeIdentityTransform();
static Float32List? _identityTransform;

Float32List _computeIdentityTransform() {
  ... your code to compute the identity matrix ...
  _identityTransform = result;
  return result;
}
```